Source/WebCore/inspector/agents/InspectorPageAgent.cpp

```diff
@@ -513,6 +514,10 @@ Inspector::Protocol::ErrorStringOr<void> InspectorPageAgent::overrideSetting(Ins
         inspectedPageSettings.setShowRepaintCounterInspectorOverride(value);
         return { };
 
+    case Inspector::Protocol::Page::Setting::SiteIsolationDebugOverlay:
+        m_client->setDeveloperPreferenceOverride(InspectorBackendClient::DeveloperPreference::SiteIsolationDebugOverlay, value);
```

Comment:
should this also modify `inspectedPageSettings`?  or does it only have an effect in the UIProcess?

Reply:
Due to site isolation, this needs to go out to UIProcess and get synced back to each web process, where the drawing actually happens.